Modules/Forum/classes/class.ilFileDataForumDrafts.php

```diff
@@ -1,20 +1,19 @@
-<?php
+<?php declare(strict_types=1);
 /* Copyright (c) 1998-2016 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 /**
  * This class handles all operations on files for the drafts of a forum object.
  *
  * @author	Nadia Matuschek <nmatuschek@databay.de>
- *
  * @ingroup ModulesForum
  */
 class ilFileDataForumDrafts extends ilFileData
 {
-    protected $obj_id = 0;
-    protected $draft_id = 0;
-    protected $drafts_path = '';
+    protected int $obj_id = 0;
+    protected int $draft_id = 0;
+    protected string $drafts_path = '';
     private $lng;
-    private $error;
+    private mixed $error;
```

Comment:
We MUST NOT use the explicit `mixed` type. The code MUST be compatible with PHP 7.4.